src/legacy/core_plugins/kibana/public/discover/np_ready/angular/response_handler.js

```diff
@@ -26,7 +26,7 @@ function tableResponseHandler(table, dimensions) {
   if (split) {
```

Comment:
Just a note: I think we can further cleanup the discover part, since it doesn't use `splitColumn`, `splitRow`

Reply:
however, doesn't need to be in this PR, just loud thinking  :)